src/views/collection-view/helpers/get-slug.ts

```diff
@@ -35,6 +36,21 @@ export function getTutorialSlug(
 	return `/${productSlug}/tutorials/${collectionFilename}/${tutorialFilename}`
 }
 
+export function appendPathWithVariant(path: string, tutorialVariant: string) {
```

Comment:
This is something we are going to shift to middleware, not going to manually update links like this.